Adds note about Figma best practices (#33)
diez/www.diez.org: docs/getting-started/figma.md

</video>
<div class="note">Shown above in the demo web app. Keep in mind that this works for iOS and Android codebases as well.</div>

### Best practices

> When using Figma and Diez together there is an important best practices worth noting: **Create a separate Figma file for your design tokens**.

Reviewer: ```suggestion
When using Figma and Diez together there are important best practices worth noting: **Create a separate Figma file for your design tokens**.
```

Author: Oh this should be "practice" not "practices". WIll push the fix.
